dts/riscv32/rv32m1_zero_riscy.dtsi

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2018 Foundries.io Ltd
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/interrupt-controller/openisa-intmux.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	aliases {
+		pcc-0 = &pcc0;
+		pcc-1 = &pcc1;
+		intmux = &intmux;
+		system-lptmr = &lptmr2;
+		pinmux-a = &pinmux_a;
+		pinmux-b = &pinmux_b;
+		pinmux-c = &pinmux_c;
+		pinmux-d = &pinmux_d;
+		pinmux-e = &pinmux_e;
+		gpio-a = &gpioa;
+		gpio-b = &gpiob;
+		gpio-c = &gpioc;
+		gpio-d = &gpiod;
+		gpio-e = &gpioe;
+		uart-0 = &uart0;
+		uart-1 = &uart1;
+		uart-2 = &uart2;
+	};
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "riscv";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x20000000 0x30000>;
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		ranges;
+
+		pcc0: clock-controller@4002b000 {
+			compatible = "openisa,rv32m1-pcc";
+			clock-controller;
+			reg = <0x4002b000 0x200>;
+			label = "PCC0";
+			#clock-cells = <1>;
+		};
+
+		pcc1: clock-controller@41027000 {
+			compatible = "openisa,rv32m1-pcc";
+			clock-controller;
+			reg = <0x41027000 0x200>;
+			label = "PCC1";
+			#clock-cells = <1>;
+		};
+
+		event1: interrupt-controller@4101f000 {
+			compatible = "openisa,rv32m1-event-unit";
+			#interrupt-cells = <1>;
+			interrupt-controller;
+			reg = <0x4101f000 0x88>;
+		};
+
+		intmux: interrupt-controller@41022000 {
+			compatible = "openisa,rv32m1-intmux";
+			#interrupt-cells = <1>;
+			interrupt-controller;
+			interrupt-parent = <&event1>;
+			interrupts = <INTMUX_CH0_IRQ>, <INTMUX_CH1_IRQ>, <INTMUX_CH2_IRQ>, <INTMUX_CH3_IRQ>, <INTMUX_CH4_IRQ>, <INTMUX_CH5_IRQ>, <INTMUX_CH6_IRQ>, <INTMUX_CH7_IRQ>;
+			reg = <0x41022000 0x20>;
+			clocks = <&pcc1 0x88>;
+			label = "INTMUX0";
+		};
+
+		/*
+		 * INTMUX channels below are somewhat arbitrary.
+		 *
+		 * The system timer (assumed at LPTMR0) is placed on channel 0,
+		 * and peripherals are in channel 1. This can be overridden with
+		 * overlays, e.g. to manage IRQ priorities, and it'll will Just
```

Comment:
```suggestion
		 * overlays, e.g. to manage IRQ priorities, and it will Just
```

(the original typo has since been fixed by @henrikbrixandersen in the RI5CY dtsi)